soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3301,7 +3521,8 @@ void DrawRandoEditor(bool& open) {
     // Shuffle Settings
     const char* randoShuffleDungeonRewards[4] = { "End of dungeons", "Any dungeon", "Overworld", "Anywhere" };
     const char* randoLinksPocket[4] = { "Dungeon Reward", "Advancement", "Anything", "Nothing" };
-    const char* randoShuffleSongs[3] = { "Song locations", "Dungeon rewards", "Anywhere" };
+    const char* randoShuffleSongs[3] = { "Song Locations", "Dungeon Rewards", "Anywhere" };
+    const char* randoShuffleTokens[4] = { "Off", "Dungeons", "Overworld", "All Tokens" };
```

Comment:
I think it just adds a tiny bit more clarity. N64 and 3DS rando also both call it specifically "All tokens". The option name isn't super long as it is either, so I don't think there's any harm in keeping it this way.

Reply:
looking at it in the menu i agree all tokens is more clear, if the option was titled "Shuffle Skulltula Tokens" having "All Tokens" would feel redundant, but since it just says "Tokensanity" it feels less redundant